stable/grafana/README.md

```diff
@@ -176,7 +177,7 @@ the url value is https://yourgerritserver/a/user%2Frepo/branches/master/files/di
 
 ## Sidecar for dashboards
 
-If the parameter `sidecar.dashboards.enabled` is set, a sidecar container is deployed in the grafana pod. This container watches all config maps in the cluster and filters out the ones with a label as defined in `sidecar.dashboards.label`. The files defined in those configmaps are written to a folder and accessed by grafana. Changes to the configmaps are monitored and the imported dashboards are deleted/updated. A recommendation is to use one configmap per dashboard, as an reduction of multiple dashboards inside one configmap is currently not properly mirrored in grafana.
+If the parameter `sidecar.dashboards.enabled` is set, a sidecar container is deployed in the grafana pod. This container watches all configmaps (or secrets) in the cluster and filters out the ones with a label as defined in `sidecar.dashboards.label`. The files defined in those configmaps are written to a folder and accessed by grafana. Changes to the configmaps are monitored and the imported dashboards are deleted/updated. A recommendation is to use one configmap per dashboard, as an reduction of multiple dashboards inside one configmap is currently not properly mirrored in grafana.
```

Comment:
To highlight that secrets are preferable, I propose replacing "watches all configmaps (or secrets)" with "watches all secrets (or configmaps, though not recommended)", plus adding a small note in a separate paragraph explaining _why_ secrets are a better choice.

Reply:
In order to stay backwards compatible I would have kept configmaps the default in the description.
But I can add a paragraph why secrets would be better. Is that a good common ground?